controllers/comments.controllers.js

```diff
@@ -0,0 +1,26 @@
+const { fetchCommentsByArticleId, insertCommentByArticleId } = require("../models/comments.models.js")
+const { checkArticleExists } = require("../models/articles.models.js")
+
+const getCommentsByArticleId = (req, res, next) => {
+    const { article_id } = req.params
+    return Promise.all([fetchCommentsByArticleId(article_id), checkArticleExists(article_id)]).then((comments) => {
```

Comment:
You may not need a checkExists function here, what happens if we pass the error to psql? If we can determine an error from the response or psql throwing an error itself it avoids us needing to run two database queries every time a patch request is called.